config/jobs/kubernetes/test-infra/test-infra-trusted.yaml

```diff
@@ -68,3 +68,38 @@ periodics:
     - name: github
       secret:
         secretName: oauth-token
+- cron: "54 * * * *"  # Every hour at 54 minutes past the hour 
+  name: ci-branchprotector
+  cluster: test-infra-trusted
+  decorate: true
+  spec: 
+    containers:
+    - name: branchprotector
+      image: gcr.io/k8s-prow/branchprotector:v20181019-08e9d55c9
+      args:
```

Comment:
TODO: this also needs to set the `command:` to match the entrypoint binary command.

Imo, this is an annoying thing to need to do manually, really our entrypoint binary hack should respect the original image `ENTRYPOINT` if `command` isn't set by the user, like k8s

Reply:
100% agree but it is a current design limitation:

* injecting the entrypoint binary as the ENTRYPOINT is how we: a) capture/upload logs from the test process and b) notify the sidecar that the test process has finished
* determining the ENTRYPOINT of a random image is non-trivial, too expensive for trigger/decorate to download every image to extract the tag whenever it schedules a job

Reply:
Technically we don't actually have to download the image, we just need the image manifest which is small, but I'm not sure if there's a nice library for this. Not to mention image pull secrets etc...

Agree this is a current design limitation, but I do think it makes it unnecessarily clunky to port "I have a container" to -> I now have a job.